cluster/manifests/event-logger/vpa.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: event-logger-vpa
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: event-logger
+spec:
+  targetRef:
+    apiVersion: apps/v1
+    kind: StatefulSet
+    name: kubernetes-event-logger
+  updatePolicy:
+    updateMode: Auto
+  resourcePolicy:
+    containerPolicies:
+    - containerName: logger
+      minAllowed:
+        memory: 100Mi
+        cpu: 20m
```

Comment:
Min can be 10m as before, no need to increase in all clusters.

Let's make the values configurable via config-item like we do for all other components.

Reply:
Done in the new commit.